testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -4068,6 +4071,7 @@ private List<DataMappingTestSetup> testDataMappingSmokeTestData()
                 .add(new DataMappingTestSetup("char(3)", "'ab'", "'zzz'"))
                 .add(new DataMappingTestSetup("varchar(3)", "'de'", "'zzz'"))
                 .add(new DataMappingTestSetup("varchar", "'łąka for the win'", "'ŻŻŻŻŻŻŻŻŻŻ'"))
+                .add(new DataMappingTestSetup("varchar", "'a \\backslash'", "'a a'")) // `a` comes after `\`
```

Comment:
```suggestion
                .add(new DataMappingTestSetup("varchar", "'a \\backslash'", "'a a'")) // `a` sorts after `\`
```

Reply:
But why is sort order important?

I feel like I'm missing lot of context regarding this test.

Reply:
the test method takes two values, the second needs to be the greater one